FSharpLu.Json/Compact.fs

```diff
@@ -28,6 +28,20 @@ module private ConverterHelpers =
     
     let getUnionCasesMemorised = memorise FSharpType.GetUnionCases
 
+    let constructUnionCaseMemorised = memorise FSharpValue.PreComputeUnionConstructor
```

Comment:
Naming suggestion: I would consider moving all the `memorized` let bindings under a `Memorised` submodule as follows:
```fs
module Memorised =
   let constructUntionCase = …
   let getUnionCase = …
```